### DISCUSSION TITLE
Question about SellYourSaaS and launching dolibarr instances...

### DISCUSSION THREAD

**Discussion by karozans** [eldy/SellYourSaas#171] [Q&A]
I finally got SYS working.  However, I am a little confused on the operation of SYS and launching specifically Dolibarr instances.  Sorry but this question is a little difficult to ask in a succinct manner.

When launching specifically Dolibarr instances on the deployment server, do I need to create a Package that creates duplicates of the Dolibarr web application under /mnt/home on the deployment servers (as described in the directions)?  Or, should I create a Package that uses the Dolibarr files on the deployment server that are located at /home/admin/wwwroot?

Why do the directions tell us to installed Dolibarr under /home/admin/wwwroot on the deployment servers if they are not used?

If the SYS software creates a duplicate of the Dolibarr software for each new instance, over time, new customers that sign up will get new software, but the customers that have been there for a long time will be down level.

I want all my customers to be on the same version of Dolibarr at all times.  If I make a change to the Dolibarr files or a custom module, I want all my customers to get the exact same changes at the same time.

Hopefully that makes sense.

Thanks

**Comment by eldy** [comment]
The application package that you declare must not deploy the files into  /home/admin/wwwroot/dolibarr because this is app of your own dolibarr and you can deploy anything. I recommand to put the files you want to deploy into  /home/admin/wwwroot/dolibarr_documents/git/myapp (dolibarr_documents is the dir to store the data and the app to deploy is data for your own dolibarr) and get the files of myapp by a git clone myapp

**Reply by karozans** [reply]
You are talking about on the admin server, right?

My question was more about the deployment server.  On the directions, it has you create files on the deployment server under /home/admin/wwwroot as well plus all the deployed instances go into /mnt/diskhome.  Is it necessary to have the files under /home/admin/wwwroot on the deployment server as well?  Don't you manage all the instances from the admin server?
